java/ql/src/semmle/code/java/frameworks/spring/Spring.qll

```diff
@@ -32,6 +32,7 @@ import semmle.code.java.frameworks.spring.SpringQualifier
 import semmle.code.java.frameworks.spring.SpringRef
 import semmle.code.java.frameworks.spring.SpringReplacedMethod
 import semmle.code.java.frameworks.spring.SpringSet
+import semmle.code.java.frameworks.spring.SpringUtil
```

Comment:
Should this import be removed?

Reply:
I think it makes sense to keep. `Spring.qll` just aims to bulk-import all of the spring-related libraries and even though `SpringUtil` currently doesn't contain any public declarations, meaning that this import doesn't do anything, then any future public additions in `SpringUtil` ought to be imported by `Spring.qll`.